### PR TITLE
Distances instead of directions

### LINKED ISSUE #159: Distances instead of directions in the track board.
See https://github.com/codingteam/pacmacs.el/issues/153#issuecomment-156943987 for more info

### PR DESCRIPTION
**Pull request by rexim**
Close #159

### CODE REVIEW

File: pacmacs.el [hunk 1]
```diff
@@ -322,46 +323,72 @@
...
+                         (-lambda ((distance-1 . _) (distance-2 . _))
+                           (< distance-1 distance-2))))
+    (pacmacs--switch-direction game-object direction)))
+
+(defun pacmacs--back-track-object (game-object)
+  (-when-let (direction (pacmacs--track-object-direction
+                         game-object
+                         (-lambda ((distance-1 . _) (distance-2 . _))
+                           (> distance-1 distance-2))))
+    (pacmacs--switch-direction game-object direction)))
```
Comment (rexim) [inline pacmacs.el]: Looks like `pacmacs--track-object` and `pacmacs--back-track-object` have a duplicate code. Just a reminder to myself to create a general function.

**Comment by ForNeVeR** [comment]
LGTM :+1:

**Comment by rexim** [comment]
Tanks @ForNeVeR! Ur da west!

**Comment by rexim** [comment]
ATT on Linux: `8ms`
ATT on Windows: `16ms`

**Pull request merged**